jlfabi-suffield.md

```diff
@@ -0,0 +1,10 @@
+## Suffield, CT
+
+### Restaurants
+- Three Figs
```

Comment:
If I went to Three Figs, what burger would you recommend and what wine pairing should I get?